src/App.tsx

```diff
@@ -1,24 +1,339 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todos } from './components/Todos';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+import {
+  getTodos, postTodos, deleteTodo, patchTodos,
+} from './api/todos';
+
+import { Todo } from './types/Todo';
+import { FilteredBy } from './types/FilteredBy';
+
+const USER_ID = 9964;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [filteredBy, setFilteredBy] = useState<FilteredBy>(FilteredBy.All);
+  const [isLoadingError, setLoadingError] = useState(false);
```

Comment:
Why do you need so many states for an error? If you can leave one boolean state that will indicate the presence of an error and a string state for the error message?